tools/sgx-lkl-docker

```diff
@@ -44,22 +44,46 @@ function build() {
 FROM ubuntu:18.04
 RUN apt-get update && apt-get install -y ca-certificates
```

Comment:
Are you using a multi-stage build to get a subset of these certificates? Is a regular Ubuntu image the best place to get these certificates?

Reply:
I'm using a multi-stage build because the second stage starts from scratch with empty base image and hence can't invoke any commands. The certificates are not a subset, it's all of them. I don't see a reason why the Ubuntu image is not a good source for getting the certificates, since that's what most containers would generally rely on.